### PR TITLE
FIX: The '-%>' closing tag should remove Windows line end characters too.

### PR DESCRIPTION
**Pull request by zoltan-berkes-epam**

### CODE REVIEW

**Comment by mde** [comment]
Please include a test, so this doesn't get regressed in the future.

File: lib/ejs.js [hunk 1]
```diff
@@ -569,7 +569,7 @@ Template.prototype = {
 
     function _addOutput() {
       if (self.truncate) {
-        line = line.replace('\n', '');
+        line = line.replace('\r', '').replace('\n', '');
```
Comment (TimothyGu) [inline lib/ejs.js]: Have you benchmarked this versus 

```js
line = line.replace(/\r|\n/g, '');
```

?

Comment (zoltan-nagy-epam) [inline lib/ejs.js]: :+1:
Your solution must be much better!

**Comment by mde** [comment]
Fixed in #122

**Pull request closed without merging**